packages/terra-demographics-banner/src/selectable-demographics-banner/SelectableDemographicsBanner.jsx

```diff
@@ -0,0 +1,170 @@
+  /**
+   * Id for Info Button.
+   */
+  infoButtonId: PropTypes.string,
+  /**
+  * Id for Selectable Tile.
+  */
+  selectableTileId: PropTypes.string,
+};
+
+const defaultProps = {
+  applicationContent: null,
+  age: undefined,
+  dateOfBirth: undefined,
+  deceasedDate: null,
+  gender: undefined,
+  gestationalAge: null,
+  identifiers: {},
+  personName: undefined,
+  photo: undefined,
+  postMenstrualAge: null,
+  preferredFirstName: null,
+  infoButtonId: undefined,
+  selectableTileId: undefined,
+};
+
+const SelectableDemographicsBanner = ({
+  age,
+  dateOfBirth,
+  gender,
+  intl,
+  personName,
+  infoButtonOnClick,
+  selectableTileOnClick,
+  infoButtonId,
+  selectableTileId,
+  ...customProps
+}) => {
+  const noDataProvided = intl.formatMessage({ id: 'Terra.demographicsBanner.noDataProvided' });
+
+  const handleKeyDown = (event) => {
+    console.log('keyspace', KEY_SPACE);
+    console.log('keyspace', KEY_RETURN);
+    if (event.keyCode === KEY_SPACE || event.keyCode === KEY_RETURN) {
+      event.preventDefault();
+    }
+  };
+
+  const handleKeyUp = (event) => {
+    if ((event.keyCode === KEY_SPACE || event.keyCode === KEY_RETURN) && selectableTileOnClick) {
+      selectableTileOnClick();
+    }
+  };
+
+  const handleOnClick = () => {
+    if (selectableTileOnClick) {
+      selectableTileOnClick();
+    }
+  };
+
+  const DemographicsBannerSelectableTile = (
+    <DemographicsBannerDisplay
+      {...customProps}
+      className={cx('selectable-demographics-banner')}
+      age={age || noDataProvided}
+      dateOfBirth={dateOfBirth || noDataProvided}
+      gender={gender || noDataProvided}
+      personName={personName || noDataProvided}
+      dateOfBirthLabel={intl.formatMessage({ id: 'Terra.demographicsBanner.dateOfBirth' })}
+      dateOfBirthFullText={intl.formatMessage({ id: 'Terra.demographicsBanner.dateOfBirthFullText' })}
+      deceasedDateLabel={intl.formatMessage({ id: 'Terra.demographicsBanner.deceased' })}
+      gestationalAgeLabel={intl.formatMessage({ id: 'Terra.demographicsBanner.gestationalAge' })}
+      gestationalAgeFullText={intl.formatMessage({ id: 'Terra.demographicsBanner.gestationalAgeFullText' })}
+      postMenstrualAgeLabel={intl.formatMessage({ id: 'Terra.demographicsBanner.postMenstrualAge' })}
+      postMenstrualAgeFullText={intl.formatMessage({ id: 'Terra.demographicsBanner.postMenstrualAgeFullText' })}
+    />
+  );
+
+  return (
+    <BackgroundTile deceasedDate={(customProps.deceasedDate)}>
+      <SelectableTile id={selectableTileId} onClick={handleOnClick} onKeyDown={handleKeyDown} onKeyUp={handleKeyUp}>
+        {DemographicsBannerSelectableTile}
+      </SelectableTile>
+      <div className={cx('divider')} />
+      <div className={cx('info-button')}>
+        {/* Needs Translation */}
+        <div className={cx('info-text')}>Info</div>
+        <Button id={infoButtonId} icon={<IconInfo />} variant="utility" isIconOnly text="info" onClick={infoButtonOnClick} />
```

Comment:
Should the text be clickable or just the icon? The icon has a small click target.

Reply:
With new changes text will also become clickable since both text and icon are wrapped within button.